packages/babel-plugin-strip-runtime/src/__tests__/extract-styles.test.ts

```diff
@@ -0,0 +1,155 @@
+import { transform } from './transform';
+
+describe('babel-plugin-strip-runtime with stylesheet extraction (extractStylesToDirectory)', () => {
+  describe('with the classic runtime', () => {
+    const runtime = 'classic';
+
+    describe('without JSX pragma', () => {
+      const code = `
+        import '@compiled/react';
+
+        const Component = () => (
+          <div css={{ fontSize: 12, color: 'blue' }}>
+            hello world
+          </div>
+        );
+      `;
+
+      it('adds styles to directory', () => {
+        const actual = transform(code, {
+          run: 'both',
+          runtime,
+          extractStylesToDirectory: { source: 'src/', dest: 'dist/' },
+        });
+
+        expect(actual).toMatchInlineSnapshot(`
```

Comment:
Is the main assertion the existence of the CSS import? Do we want to assert any specific CSS output? Could they regress?

Reply:
this test was added in https://github.com/atlassian-labs/compiled/pull/1513/files#diff-a3198cfeec7834168f1ae1a2e61d0f416af2b7167289b727aa4a260d93f59579 i think

it's testing both

* `app.compiled.css` import was added
* css styling no longer exists in any form in the babel output

it doesn't look like the contents of `app.compiled.css` is tested (@liamqma can you confirm?), maybe i can add it?

Reply:
this test was added in https://github.com/atlassian-labs/compiled/pull/1513/files#diff-a3198cfeec7834168f1ae1a2e61d0f416af2b7167289b727aa4a260d93f59579 i think

it's testing both

app.compiled.css import was added
css styling no longer exists in any form in the babel output
it doesn't look like the contents of app.compiled.css is tested (@liamqma can you confirm?), maybe i can add it?